automation/taskcluster/rename_artifacts.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is to eventually replace its `rename_snapshot_artifacts` sibling. Since these two need to co-exist for a while, I think we're okay to live with the code duplication, since it's short lived.